sematic/ui/src/utils.tsx

```diff
@@ -59,9 +60,23 @@ export function fetchJSON({
 }
 
 export function useLogger() {
-  const devLogger = useMemo(() => process.env.NODE_ENV === "development" ?
-    (...args: any[]) => console.log("DEV DEBUG: ", ...args) :
-    () => { }, []);
+  const { search } = useLocation();
+
+  const isLoggingExplicitlyTurnedOn = useMemo(
+    () => (new URLSearchParams(search)).has('debug'), [search]);
```

Comment:
Please add this new feature to the changelog. It's not immediately visible from the PR title, and we'll forget to include it during the release procedure.

Reply:
Great suggestion. Thanks! Added to changelog file.